vendor_unpack.sh

```
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+#unzip vendors into folder
+cd Assets/vendor
+unzip ../../vendor.zip
+
+#delete plugins/fmod if exists
+rm -r ../Plugins/FMOD 2>/dev/null
+
+#move vendor/fmod into plugins/fmod
+mv FMOD ../Plugins/FMOD
+
+#remove files if exist
+rm -f ../Plugins/FMOD/Resources/FMODStudioSettings.asset{,.meta}
```

Please could you add a `2>/dev/null` here to suppress the error message if the files don't exist?

---

Actaully, I've just noticed you've used `-f` which will hide the message anyway. Can you remove the `2>/dev/null` and add `-f` instead on the previous `rm` command?